wicker/core/datasets.py

```diff
@@ -22,6 +23,95 @@
 FILE_LOCK_TIMEOUT_SECONDS = 300
 
 
+def get_file_size_s3_multiproc(buckets_keys: List[Tuple[str, str]]) -> int:
+    """Get file size of s3 files, most often column files.
+
+    This works on any list of buckets and keys but is generally only
+    used for column files as those are the majority of what is stored on
+    s3 for Wicker. Wicker also stores parquet files on s3 but those are limited
+    to one file per dataset and one schema file.
+
+    This splits your buckets_keys_list across multiple processes on your local host
+    where each process is then multi threaded further. This reduces the i/o wait by
+    parellelizing across all available procs and threads on a single machine.
+
+    Args:
+        buckets_keys: (List[Tuple[str, str]]): A list of buckets and keys for which
+        to fetch size in bytes on s3. Tuple index 0 is bucket and index 1 is key of the file.
+
+    Returns:
+        int size of file list in bytes.
+    """
+    buckets_keys_chunks = chunk_data_for_split(chunkable_data=buckets_keys, chunk_number=200)
+
+    logging.info("Grabbing file information from s3 heads")
+    pool = Pool(cpu_count() - 1)
+    return sum(list(tqdm.tqdm(pool.map(get_file_size_s3_threaded, buckets_keys_chunks))))
```

Comment:
Shouldn't we disable `tqdm` by default?! Concerned that this would pollute the logging in cloud environments. Likewise elsewhere.

Reply:
I was only going to disable it in the copy to gcloud PR. I can't see this used anywhere in a cloud env until that PR is in. IMO there's almost no use for this function on cloud right now.

Reply:
@pickles-bread-and-butter imo maybe we should just nuke the usage of `tqdm` here since we're going to be nuking it anyways

Reply:
I'll nuke it, I definitely prefer having it instead of it churn less visibly but we can add it back if needed.

Reply:
sorry not yet addressed?

Reply:
Sorry forgot to push it up, it's pushed now